vclib-openid/src/commonTest/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopIsoProtocolTest.kt

```diff
@@ -99,6 +100,7 @@ class OidcSiopIsoProtocolTest : FreeSpec({
             OidcSiopVerifier.RequestOptions(
                 representation = ConstantIndex.CredentialRepresentation.ISO_MDOC,
                 credentialScheme = ConstantIndex.AtomicAttribute2023,
+                requestedAttributes = listOf("given-name"),
```

Comment:
why not use a constant here?

Reply:
AtomicAttribute2023 does not specify any attribute constants